Roslyn-SDK.sln

```diff
@@ -20,6 +20,10 @@ EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Roslyn.SyntaxVisualizer.Extension", "src\VisualStudio.Roslyn.SDK\SyntaxVisualizer\Roslyn.SyntaxVisualizer.Extension\Roslyn.SyntaxVisualizer.Extension.csproj", "{BF3477AA-22C9-4751-A4C0-01EF30728AE9}"
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Microsoft.CodeAnalysis.Testing", "Microsoft.CodeAnalysis.Testing", "{A3E08CE3-2358-4D36-875B-82C99AC61CD8}"
+	ProjectSection(SolutionItems) = preProject
+		src\Microsoft.CodeAnalysis.Testing\Directory.Build.props = src\Microsoft.CodeAnalysis.Testing\Directory.Build.props
+		src\Microsoft.CodeAnalysis.Testing\Directory.Build.targets = src\Microsoft.CodeAnalysis.Testing\Directory.Build.targets
+	EndProjectSection
```

Comment:
💡 These don't need to be added to the solution. They are visible in Solution Explorer with this button:

![image](https://user-images.githubusercontent.com/1408396/135881173-ad24d3be-a2fd-47e6-b4f7-32631d8728c2.png)

Reply:
It's not about viewing the file, it's about getting FindInFiles with scope solution working as expected. (for guys like me being not so used to all the features and tweaks in an unknown solution)
However I won't insist in adding this...

Reply:
I find it's always a losing battle to keep solution files synchronized with the solution. The only way it's _always_ consistent is to omit them all and point users to the solution filter button. 😄